pkgs/development/python-modules/importlib-metadata/default.nix

```diff
@@ -8,22 +8,23 @@
 , configparser
 , isPy3k
 , importlib-resources
+, toml
 , packaging
 }:
 
 buildPythonPackage rec {
   pname = "importlib-metadata";
-  version = "1.7.0";
+  version = "3.4.0";
 
   src = fetchPypi {
     pname = "importlib_metadata";
     inherit version;
-    sha256 = "10vz0ydrzspdhdbxrzwr9vhs693hzh4ff71lnqsifvdzvf66bfwh";
+    sha256 = "sha256-+l2qRHenQUrjTpWULk3Qf2Kt9YkUPIdcEzweU8Tv840=";
   };
 
   nativeBuildInputs = [ setuptools_scm ];
 
-  propagatedBuildInputs = [ zipp ]
+  propagatedBuildInputs = [ toml zipp ]
     ++ lib.optionals (!isPy3k) [ pathlib2 contextlib2 configparser ];
```

Comment:
This version no longer support python 2.7.

Reply:
Removed, thanks.